apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -30,6 +31,34 @@ export const EventSetupTab = (
   const [showLocationModal, setShowLocationModal] = useState(false);
   const [editingLocationType, setEditingLocationType] = useState<string>("");
   const [selectedLocation, setSelectedLocation] = useState<OptionTypeBase | undefined>(undefined);
+  const [multipleDuration, setMultipleDuration] = useState(eventType.metadata.multipleDuration);
+
+  const multipleDurationOptions = [
```

Comment:
Couldn't find a better place to have this constant array of available durations for multiple duration event types, suggestions?